tests/algorithms/test_dagger.py

```diff
@@ -291,31 +257,31 @@ def add_samples():
     trainer.extend_and_update(dict(n_epochs=1))
 
 
-def test_trainer_makes_progress(init_trainer_fn, cartpole_venv, cartpole_expert_policy):
+def test_trainer_makes_progress(init_trainer_fn, pendulum_venv, pendulum_expert_policy):
     with torch.random.fork_rng():
         # manually seed to avoid flakiness
         torch.random.manual_seed(42)
-        cartpole_venv.action_space.seed(42)
+        pendulum_venv.action_space.seed(42)
 
         trainer = init_trainer_fn()
         pre_train_rew_mean = rollout.mean_return(
             trainer.policy,
-            cartpole_venv,
+            pendulum_venv,
             sample_until=rollout.make_min_episodes(15),
             deterministic_policy=False,
         )
         # note a randomly initialised policy does well for some seeds -- so may
-        # want to remove this check if changing seed.
-        assert pre_train_rew_mean < 100
-        # Train for 4 iterations. (3 or less causes test to fail on some configs.)
-        for i in range(4):
+        # want to adjust this check if changing seed.
```

Comment:
May help to add some context:

```suggestion
        # want to adjust this check if changing seed. Pendulum return can range
        # from -1,200 to -130 (approx.), per Figure 3 in this PDF (on page 3):
        # https://arxiv.org/pdf/2106.09556.pdf
```